DiceKeys/Views/Screens/EditableDiceKeyView/KeyboardCommands.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  KeyboardCommands.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 22/01/21.
+//
+
+import SwiftUI
+
+struct KeyboardCommandsModel: Identifiable {
```

Comment:
This isn't really a model.  It's a mapping.  May I suggest

```
   let idToKeyEquivalent: [String: KeyEquivalent] = {
      var idToKeyEquivalentBuilder: [String: KeyEquivalent] = [
         "delete": .delete,
         ...
      ]
      for (letter in FaceLetters) { idToKeyEquivalent[letter] = KeyEquivalent(letter.rawValue.first!)
      for (digit in FaceDigits) { idToKeyEquivalent[digit] = KeyEquivalent(digit.rawValue.first!)
      return idToKeyEquivalentBuilder
   }()
```

To iterate through this to build your view:
```
            ForEach(idToKeyEquivalent.sorted(by: >), id: \.key) { key, value in
...
            }
```

Reply:
Will update this, push code again for the same.